tests/dmctl_command/run.sh

```diff
@@ -51,6 +51,8 @@ function run() {
         exit 1
     fi
 
+    run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
```

Comment:
ditto

Reply:
Now we check master connection in dmctl. So we must run dm master before run dmctl.